### ISSUE TITLE
False positive for leading newlines for source map/copyright comments in comment-whitespace-inside reports

### ISSUE THREAD

**Issue by jwilsson** [stylelint/stylelint#2120]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

A bug where `comment-whitespace-inside` reports source map/copyright comments with leading new lines.

> Which rule, if any, is this issue related to?

`comment-whitespace-inside`

> What CSS is needed to reproduce this issue?

```css
/*!
Some stuff...
*/
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "comment-whitespace-inside": "always"
  }
}
```

> Which version of stylelint are you using?

`7.6.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

Via Gulp plugin, but it's reproducible via CLI as well, e.g.

```shell
stylelint --config config.json style.css
```

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

No, it's regular CSS.

> What did you expect to happen?

No warnings to be flagged since "regular" comments allow new lines, e.g.

```css
/*
Some stuff...
*/
```

is fine.

> What actually happened (e.g. what warnings or errors you are getting)?

The following warnings were flagged:

```shell
style.css
 1:3  ✖  Expected whitespace after "/*"   comment-whitespace-inside
```

I'd be happy to submit a PR fixing it!

**Comment by evilebottnawi** [comment]
@jwilsson Thanks for feedback! Go PR :smile:

**Comment by davidtheclark** [comment]
Closed by #2121.
